### PR TITLE
Remove code from sender account

### PR DESCRIPTION
**Pull request by MariusVanDerWijden**
In the future, we would like to disallow the sender from having code in the EOA sending account

TODO

- [ ] Add negative testcase fillers
- [x] Regenerate tests

### CODE REVIEW

**Comment by winsvega** [comment]
looks good so far.

**Comment by MariusVanDerWijden** [comment]
@winsvega QQ: to get this merged I just need to fill the tests right? (and rebase on master)

**Comment by winsvega** [comment]
Yes. I can help with that

**Comment by holiman** [comment]
Can this be merged? Would be good to have it in so we can merge the change

**Comment by MariusVanDerWijden** [comment]
I would like to create a negative test example in an upcoming PR (not in this one)

**Comment by qbzzt** [comment]
> I would like to create a negative test example in an upcoming PR (not in this one)

You mean a test that is supposed to fail? The way you do that is you add an `expectExcept:` field. See https://ethereum-tests.readthedocs.io/en/latest/state-transition-tutorial.html#tests-that-are-supposed-to-fail

**Comment by winsvega** [comment]
don't forget --fillchain

File: src/GeneralStateTestsFiller/stCreate2/create2collisionBalanceFiller.json [hunk 1]
```diff
@@ -106,7 +106,7 @@
         "pre" : {
             "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
                 "balance" : "1000000000000000000",
-                "code" : "0x",
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stCreate2/create2collisionBalanceFiller.json:109]: code 0x means empty. there is no need to make this change

**Comment by winsvega** [comment]
ready?

**Comment by holiman** [comment]
> ready?

Is that a question for Marius? I think it would be great if this PR could be merged now, unless there's something more needed to be done with it from _your_ perspective (?)

**Comment by winsvega** [comment]
https://github.com/ethereum/tests/pull/893

**Pull request closed without merging**